test/hotspot/jtreg/ProblemList.txt

```diff
@@ -91,6 +91,7 @@ gc/stress/TestStressG1Humongous.java 8286554 windows-x64
 
 
 runtime/jni/terminatedThread/TestTerminatedThread.java 8219652 aix-ppc64
+runtime/jni/nativeStack/TestNativeStack.java 8311092 linux-arm
```

Comment:
When you ProblemList a test, the bugid you enter is for the bug that will fix the test (or product) and so allow you to remove it from the ProblemList again. If you want this test permanently disabled on ARM 32-bit then that should be done in the test via an `@requires` clause.

Thanks.

Reply:
Thank you!!! Updated PR to include requires condition rather than ProblemList entry